Lib/ufo2ft/makeotfParts.py

```diff
@@ -180,30 +180,4 @@ def setupFile_featureTables(self):
                     self.outline[tag] = table
 
         elif self.features.strip():
-            if self.font.path is not None:
-                self.features = forceAbsoluteIncludesInFeatures(self.features, self.font.path)
-            fd, fea_path = tempfile.mkstemp()
-            with open(fea_path, "w") as feafile:
-                feafile.write(self.features)
-            addOpenTypeFeatures(fea_path, self.outline)
-            os.close(fd)
-            os.remove(fea_path)
-
-includeRE = re.compile(
-    "(include\s*\(\s*)"
-    "([^\)]+)"
-    "(\s*\))" # this won't actually capture a trailing space.
-    )
-
-
-def forceAbsoluteIncludesInFeatures(text, directory):
-    for match in reversed(list(includeRE.finditer(text))):
-       start, includePath, close = match.groups()
-       # absolute path
-       if os.path.isabs(includePath):
-           continue
-       # relative path
-       srcPath = os.path.normpath(os.path.join(directory, includePath.strip()))
-       includeText = start + srcPath + close
-       text = text[:match.start()] + includeText + text[match.end():]
-    return text
+            addOpenTypeFeatures(self.outline, self.font.path, self.features)
```

Comment:
relative include paths are relative to the directory of the including file. That is, if an include statement is present, then the feaLib lexer will append the included filename to the directory of the including file. If you set the `fea_path` argument to the path of the UFO, then the relative includes' directory will be the directory where the UFO itself is.